src/main/java/com/ip/tasksmanager/Duke.java

```diff
@@ -0,0 +1,144 @@
+    }
+
+    /**
+     * Prints out the taskType, status, taskName and taskDate of the task added by the user
+     * and prints the current total number of tasks in the user's list
+     * Add to total taskCount (since new task is added)
+     */
+    public static void printAddedTask() {
+        System.out.println("  " + tasks[taskCount].convertToTaskOutputString()); //prints task added
+        taskCount++;
+        //prints current total number of tasks (in the list of tasks):
+        System.out.print("Now you have " + taskCount + " task");
+        if (taskCount > 1) {
+            System.out.print("s");
+        }
+        System.out.println(" in the list.");
+    }
+
+    public static void main(String[] args) {
+
+        saysHiToUser();
+
+        Scanner in = new Scanner(System.in);
+        String input = in.nextLine(); //take in User's first input:
+        //list, mark or add user's tasks (until user inputs "bye"):
+        while (!input.equals("bye")) {
+
+            System.out.println(line); //start of current response to User
+
+            //add and store task to list of tasks
+            if (input.startsWith("todo")) {
+                System.out.println("Got it. I've added this task:");
+
+                separateTypeOfTaskAndTaskInputString(input);
+                taskName = taskInputString;
+
+                tasks[taskCount] = new ToDos(taskName); //add task to list
+                printAddedTask();
+            } else if (input.startsWith("deadline")) {
+                System.out.println("Got it. I've added this task:");
+
+                separateTypeOfTaskAndTaskInputString(input);
+                splitTaskNameAndDate(taskInputString);
+
+                tasks[taskCount] = new Deadlines(taskName, taskDate); //add task to list
+                printAddedTask();
+            } else if (input.startsWith("event")) {
+                System.out.println("Got it. I've added this task:");
+
+                separateTypeOfTaskAndTaskInputString(input);
+                splitTaskNameAndDate(taskInputString);
+
+                tasks[taskCount] = new Events(taskName, taskDate); //add task to list
+                printAddedTask();
+            }
+            //OR: lists all the user's current tasks in the format of taskType,taskStatus,taskName(and taskDate):
+            else if (input.startsWith("list")) {
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < taskCount; i++) {
+                    int taskNumber = i+1; //stores the current numbering of the bulleted tasks
+                    System.out.println(taskNumber + "." + tasks[i].convertToTaskOutputString());
+                }
+            }
+            //OR: mark current task as 'done' & outputs the taskType,taskStatus,taskName(and taskDate):
+            else if (input.startsWith("done")) {
+                String[] commandAndTaskNumber = input.split(" ");
```

Comment:
Very logical action here. Perhaps you could consider using "commandAndTaskNumbers" (plural forms are used on names representing a collection of objects) as the variable name after splitting input? :D